src/main/java/duke/DukeBlankDescriptionsException.java

```diff
@@ -0,0 +1,4 @@
+package duke;
+
+public class DukeBlankDescriptionsException extends Exception {
+}
```

Comment:
Consider creating packages for your different exceptions. Also applies to your other classes as well that may fit under a category

Reply:
Thank you. Will change it in future commits.